lib/msf/core/module.rb

```diff
@@ -278,6 +278,14 @@ def fail_with(reason, msg=nil)
     raise RuntimeError, "#{reason.to_s}: #{msg}"
   end
 
+  # Whether {user_data} contains everything necessary to make a
+  # `MetasploitDataModels::AutomaticExploitation::MatchResult`
+  #
+  # @return [bool]
+  def user_data_is_match?
```

Comment:
However, since it's not really an object your looking for, would something like is_valid? Or maybe is_sufficient?  Meh. 
Ooh, ooh, suffices? ;)